packages/core-utils/src/httpie.ts

```diff
@@ -18,14 +17,18 @@ export class HttpieError extends Error {
         });
 
         if (error.response) {
-            Object.defineProperty(this, "response", {
-                enumerable: false,
-                value: {
-                    body: parseJSON(error.response.body).value,
-                    headers: error.response.headers,
-                    status: error.response.statusCode,
-                },
-            });
+            try {
+                Object.defineProperty(this, "response", {
+                    enumerable: false,
+                    value: {
+                        body: JSON.parse(error.response.body || {}),
```

Comment:
```suggestion
                        body: JSON.parse(error.response.body || "{}"),
```

`JSON.parse` expects a string, not an object, otherwise you'll get a `SyntaxError` as it will try to parse `[object Object]` from `{}.toString()`.